app/lib/efile/nj/nj_retirement_income_helper.rb

```diff
@@ -1,27 +1,51 @@
 module Efile
   module Nj
     class NjRetirementIncomeHelper
+
+      attr_accessor :non_military_1099rs
       def initialize(intake)
         @intake = intake
+        @non_military_1099rs = @intake.state_file1099_rs.select do |state_file_1099r|
```

Comment:
This is duplicative logic from the NJ calculator, for now. I made a note in the story to do a fast-follow refactor to remove this from the calculator and ONLY reference the logic here in the helper instead for this.

Because of the urgency of getting this work tested, and because that refactor will require a bunch of updates to tests as well, figured it could be a fast-follow.